### PR TITLE
Rework Ecosystem section

### PR DESCRIPTION
**Pull request by SebastianSchildt**
The Ecosystem section in the official documentation was largely outdated, i.e. it mentions the dormant IoT Event Analytics, it contained a bit outdated information on KUKSA, and probably worse, a lot of things using VSS nowadays not even been mentioned.

While the texts could likely be better as well (please comment), the main thing this PR does, is suggesting a slightly different structure:

 - Still mention VSS-tools as the set "of core" COVESA-supported tools
 - Instead of listing individual software solutions (and then needing to update it often, getting into discussion whether to "update" there, or what maturity the isolated thing should have, ...), I just put a generic page linking to the COVESA wiki, which is updated more easily, and where people than also might find the Spotlight videos, AMM talks etc
 - I created a section about related "standards", that currently encompass the W3C one. As this are are not "just" products, and probably are not evolving, proliferating as much as (Open Source) software, I though it might be useful to keep them. In any case would be good if the text and links (just copied from the old page) for VISS is still ok @UlfBj   and for VSSo maybe @jdacoello may take a look/suggest better two sentences. I think on neither we should get too wordy 8we have the links), but it should give the right idea.

Screenshots for convenience:

image

image

image

image

### CODE REVIEW

**Comment by UlfBj** [comment]
I would like the following VISS related updates:

- [Vehicle Information Service Specification (VISS) V1](https://www.w3.org/TR/vehicle-information-service/), published as a Discontinued Draft.

- [Vehicle Information Service Specification (VISS) V2](https://github.com/w3c/automotive/), feature complete and in process to enter the Candidate Recommendation stage.

File: docs-gen/content/ecosystem/tools.md [hunk 1]
```diff
@@ -1,15 +1,16 @@
 ---
-title: "Tools"
-date: 2021-07-04T13:27:46+0000
+title: "Offcial VSS-Tools"
```
Comment (erikbosch) [inline docs-gen/content/ecosystem/tools.md:2]: Spelling error: "Offcial"

File: docs-gen/content/ecosystem/tools.md [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
+VSS can be transformed into other formats or generated as code to integrate with other tooling environments and ecosystems.
 
-Fig 1 shows the schematics of the top-level process.
+The COVESA [vss-tools project](https://github.com/covesa/vss-tools) contains various helpful converters and is also used to validate the syntactic validity of the VSS standard catalogue. 
+
+If new features are introduced to the VSS specification, the community will ensure they are supported in the official vss-tools as well.
 
 ![Signal tree](/vehicle_signal_specification/images/multi_target.png)<br>
-*Fig 1. Generating documents from specification*
+*Fig 1. Generating various output formats from a VSS specification*
```
Comment (erikbosch) [inline docs-gen/content/ecosystem/tools.md:14]: If you have time, modernizing/recreating this picture would be great. We do not have a markdown generator, and the names of the *.vspec files shown does not match "real files". Maybe better to make it more general, like showing one box for "Standard catalog" and one or more boxes for overlays.

File: docs-gen/content/ecosystem/tools.md (hunk 1 above)
Comment (SebastianSchildt) [reply docs-gen/content/ecosystem/tools.md]: Done

File: docs-gen/content/ecosystem/tools.md (hunk 2 above)
Comment (SebastianSchildt) [reply docs-gen/content/ecosystem/tools.md:14]: I will look into it

**Comment by SebastianSchildt** [comment]
> I would like the following VISS related updates:
> 
> * [Vehicle Information Service Specification (VISS) V1](https://www.w3.org/TR/vehicle-information-service/), published as a Discontinued Draft.
> * [Vehicle Information Service Specification (VISS) V2](https://github.com/w3c/automotive/), feature complete and in process to enter the Candidate Recommendation stage.

Done

**Comment by erikbosch** [comment]
Meeting notes:

- OK to merge
- Image can be updated later, not urgent
